zds/member/forms.py

```diff
@@ -48,7 +48,9 @@ class LoginForm(AuthenticationForm):
             "Vous n’avez pas encore activé votre compte, vous devez le faire pour pouvoir vous connecter sur le site."
             " <a href={}>Vous n’avez pas reçu le courriel d'activation ?</a>"
         ),
-        "banned": _("Vous n’êtes pas autorisé à vous connecter sur le site, vous avez été banni par un modérateur."),
+        "banned": _(
+            "Vous n’êtes pas autorisé à vous connecter sur le site, vous avez été banni par un modérateur pour la raison suivante : "
+        ),
```

Comment:
```suggestion
        "banned": _(
            "Vous n’êtes pas autorisé à vous connecter sur le site. Vous avez été banni par un modérateur pour la raison suivante : « {} »."
        ),
```

J'ajoute un espace réservé pour insérer la raison, c'est plus propre (notamment si on rechange la phrase un jour).

Faire deux phrases est plus lisible et un peu de ponctuation/typographie (espaces insécables notamment) pour faire un peu propre à pas cher.